conf.d/bin/git-forgit

```diff
@@ -0,0 +1 @@
+../../bin/git-forgit
```

Comment:
I'm curious what the addition of this new file does. @carlfriedrich, can you explain?

Reply:
@wren The fish plugin manager [fisher](https://github.com/jorgebucaran/fisher) only installs files in [these three folders](https://github.com/jorgebucaran/fisher#creating-a-plugin):

- `completions`
- `conf.d`
- `functions`

So symlinking `bin/git-forgit` to `conf.d/bin/git-forgit` was the only way I found to actually install the `git-forgit` script for fish.

@cjappl Maybe you know more about the fish plugin architecure than me and might have a better idea? Having a `bin` subdir in your `conf.d` directory surely seems a bit unorthodox. I found that it works, though, and couldn't come up with a cleaner idea.

Reply:
Great question...

I unfortunately have no knowledge of fisher/fish plugins. You may be able to `git blame` and hunt down the original person who implemented that and ask them. I didn't contribute to the plugin-ification of forgit